tensorpipe/test/channel/cuda_ipc/cuda_ipc_test.cc

```diff
@@ -17,70 +17,23 @@
 
 namespace {
 
-class CudaWrapper : public DataWrapper {
+class CudaChannelTestHelper : public ChannelTestHelper<tensorpipe::CudaTensor> {
```

Comment:
Shouldn't this be called `CudaIpcChannelTestHelper`? (Note the `Ipc`)

Reply:
Good catch, will update.